bitnami/apache/values.yaml

```diff
@@ -224,9 +224,18 @@ command: []
 ## @param args Override default container args (useful when using custom images)
 ##
 args: []
-## @param lifecycleHooks for the Apache server container(s) to automate configuration before or after startup
-##
-lifecycleHooks: {}
+## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/
+## ref: https://httpd.apache.org/docs/2.4/en/stopping.html#gracefulstop
+## @param lifecycleHooks.preStop.exec.command Override default apache preStop command
+##
+lifecycleHooks:
+  preStop:
+    exec:
+      command:
+        - kill
+        - -s
+        - SIGWINCH
+        - "1"
```

Comment:
The process 1 is httpd, but it can me changed by

`$(cat /opt/bitnami/apache/var/run/httpd.pid)`